libs/application-generic/src/usecases/upsert-control-values/upsert-control-values.command.ts

```diff
@@ -1,9 +1,8 @@
 import { IsNotEmpty, IsObject, IsOptional, IsString } from 'class-validator';
-import { JsonSchema } from '@novu/framework';
 import { NotificationStepEntity } from '@novu/dal';
-import { OrganizationLevelCommand } from '../../commands';
+import { EnvironmentCommand } from '../../commands';
 
-export class UpsertControlValuesCommand extends OrganizationLevelCommand {
+export class UpsertControlValuesCommand extends EnvironmentCommand {
```

Comment:
The incorrect base command was used here, which resulted in `environmentId` being an optional type when it should have been required.